.changeset/nice-books-tease.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Just like [here](https://github.com/FuelLabs/fuels-ts/pull/3089#discussion_r1878869571) and [here](https://github.com/FuelLabs/fuels-ts/pull/3249#discussion_r1878876211), please add a task list of everything pending in the PR description. I saw you added a comment down there, but please move it up to the description. We can edit it and wipe it off when it's done.

Reply:
Already did :)

<img width="834" alt="image" src="https://github.com/user-attachments/assets/99a4e64e-8e12-47ad-b35e-6425ca52bd14">

Reply:
My bad, I did not realize it was identical to [this](https://github.com/FuelLabs/fuels-ts/pull/3402#pullrequestreview-2484468139).